pygeo/DVConstraints.py

```diff
@@ -237,8 +237,8 @@ def setSurface(self, surf, name='default', addToDVGeo=False, DVGeoName='default'
         >>> DVCon.setSurface(surf)
 
         """
-        if name in self.surfaces.keys():
-            raise KeyError('Surface names must be unique. Repeated surface name: ' + str(name))
+        # if name in self.surfaces.keys():
+        #     raise KeyError('Surface names must be unique. Repeated surface name: ' + str(name))
```

Comment:
Do we want to remove these lines? We don't check this anymore?

Reply:
oh oops, That's a hack that was supposed to stay on my fork.